app/models/sale.rb

```diff
@@ -168,6 +169,12 @@ def accounts_receivable_reverse_event
     end
   end
 
+  def report_delivery(transition)
+    if transition.args[1] == '1'
```

Comment:
Can this be described in a comment.

For one that only looks at the code "transition.args[1]" doesn't mean anything.